README.md

```diff
@@ -247,45 +274,6 @@ with the `allennlp` command (whether you installed from `pip` or from source).
 `allennlp` has various subcommands such as `train`, `evaluate`, and `predict`.
 To see the full usage information, run `allennlp --help`.
 
-## Docker images
-
-AllenNLP releases Docker images to [Docker Hub](https://hub.docker.com/r/allennlp/) for each release.  For information on how to run these releases, see [Installing using Docker](#installing-using-docker).
-
-### Building a Docker image
```

Comment:
Why did this go away?

Reply:
I just moved it into the "Installing via Docker" section.

Reply:
You did 🙈